iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/AbstractDeclaration.java

```diff
@@ -19,15 +19,16 @@
  */
 package org.sonar.iac.arm.tree.impl.bicep;
 
+import java.util.List;
+
 import org.sonar.iac.arm.tree.api.Expression;
 import org.sonar.iac.arm.tree.api.Identifier;
+import org.sonar.iac.arm.tree.api.bicep.HasKeyword;
 import org.sonar.iac.arm.tree.api.bicep.SyntaxToken;
 import org.sonar.iac.arm.tree.impl.AbstractArmTreeImpl;
 import org.sonar.iac.common.api.tree.Tree;
 
-import java.util.List;
-
-public abstract class AbstractDeclaration extends AbstractArmTreeImpl {
+public abstract class AbstractDeclaration extends AbstractArmTreeImpl implements HasKeyword {
```

Comment:
Why does this abstract class not implement the `Declaration` interface instead of the `HasKeyword`

Reply:
Good catch!